### ISSUE TITLE
Tornado 4.5.x fails with deflate socket

### ISSUE THREAD

**Issue by ramonz** [nsqio/pynsq#198]
Tornado 4.5.2 uses memoryview slicing when writing to fd. That causes the exception in deflate socket object

`/local/lib/python2.7/site-packages/tornado/iostream.py:862`
```
     num_bytes = self.write_to_fd(
         memoryview(self._write_buffer)[start:start + size])
```

```
ERROR:nsq.async:uncaught exception in data event
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/nsq/async.py", line 287, in _read_body
    self.trigger(event.DATA, conn=self, data=data)
  File "/usr/local/lib/python2.7/dist-packages/nsq/event.py", line 85, in trigger
    ev(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/nsq/async.py", line 501, in _on_data
    self.trigger(event.RESPONSE, conn=self, data=data)
  File "/usr/local/lib/python2.7/dist-packages/nsq/event.py", line 85, in trigger
    ev(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/nsq/async.py", line 464, in _on_response_continue
    self.trigger(event.READY, conn=self)
  File "/usr/local/lib/python2.7/dist-packages/nsq/event.py", line 85, in trigger
    ev(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/nsq/reader.py", line 491, in _on_connection_ready
    conn.send(protocol.subscribe(self.topic, self.channel))
  File "/usr/local/lib/python2.7/dist-packages/nsq/async.py", line 293, in send
    self.stream.write(data)
  File "/usr/local/lib/python2.7/dist-packages/tornado/iostream.py", line 406, in write
    self._handle_write()
  File "/usr/local/lib/python2.7/dist-packages/tornado/iostream.py", line 862, in _handle_write
    memoryview(self._write_buffer)[start:start + size])
  File "/usr/local/lib/python2.7/dist-packages/tornado/iostream.py", line 1064, in write_to_fd
    return self.socket.send(data)
  File "/usr/local/lib/python2.7/dist-packages/nsq/deflate_socket.py", line 42, in send
    chunk = self._compressor.compress(data)
TypeError: must be string or read-only buffer, not memoryview
```

Prev Tornado 4.4.x works fine

**Comment by ramonz** [comment]
pynsq==0.8.1

**Comment by mreiferson** [comment]
Fixed in #195, which hasn't been released.
